cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/local/DataStorageImpl.kt

```diff
@@ -47,6 +49,14 @@ private class DataStorageImpl(
         PreferenceManager.getDefaultSharedPreferences(context)
     }
 
+    override var localDataVersion: Int
+        get() = preference
+            .getInt(KEY_LOCAL_DATA_VERSION, 1)
```

Comment:
The correct value for when the key is not found should be 0, so we make sure returning users will call `consent-stauts` at least once (and therefore get GPP data).

```kotlin
  .getInt(KEY_LOCAL_DATA_VERSION, 0)
```

Reply:
makes sense, wasn't sure if we want to enforce users to get GPP data right after we release it